Sources/Commands/SwiftTool.swift

```diff
@@ -1128,6 +1128,8 @@ extension Basics.Diagnostic {
             writer.write("error: ", inColor: .red, bold: true)
         case .warning:
             writer.write("warning: ", inColor: .yellow, bold: true)
+        case .output:
+            writer.write("output: ", inColor: .gray, bold: true)
```

Comment:
I wonder if output should be prefix free

Reply:
That's a good point — it was the same with output from command plugins, initially that we prefixed and it got annoying.